### PR TITLE
Add Python 3.11 to the testing

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,6 +33,7 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+          cache: 'poetry'
```
Comment (browniebroke) [inline .github/workflows/ci.yml:36]: That broke CI a few month ago while not providing a significant speed increase. I decided to remove it: https://github.com/browniebroke/django-codemod/pull/658

```suggestion
```

**Review by browniebroke — APPROVED** [review]
Thanks, LGTM

**Comment by cclauss** [comment]
Do the Python 3.11 performance gains matter for the process of using this tool?

**Comment by browniebroke** [comment]
Probably... Although I haven't profiled where the bottlenecks are, my guess is that it might be a mix of IO and CPU.

If you're concerned about speed and don't need Python 3.7, I would encourage you to check out django-upgrade which is _much_ faster and has more features.

**Pull request merged**